### PR TITLE
Resin doors now have the same armour values as resin walls

### PR DESCRIPTION
**Pull request by MLGTASTICa**
## About The Pull Request
Gives resin doors the same armour values as resin walls (60% bullet and laser resist)
They previously did not have any armor at all
Labeled as balance because im not sure if its a fix or not , and if it was intended for them to be this weak.

## Why It's Good For The Game
Even if they're meant to be weaker, the fact that they had less integrity than walls and no armour made it obvious why no xeno would use them, 4 P90 shots would take down a door, making them meaningless.
They're still far weaker compared to a wall, but now they can resist small arms fire way better in general(just like walls)
## Changelog
:cl:
balance: Resin doors now have the same armour as walls.
/:cl:

### CODE REVIEW

**Comment by theta060** [comment]
make a maze where xenos can fully walk into marines can be stuck and for it to take the same hits to break as a normal wall maze?

**Comment by theta060** [comment]
> > make a maze where xenos can fully walk into marines can be stuck and for it to take the same hits to break as a normal wall maze?
> 
> Doors need adjaecent resin walls as support , ontop of it , doors do not regenerate like walls do (and they don't get a health bonus the earlier they were built either). As a marine , you could also not rush into the maze and instead wait for flamethrowers or MGs to take care of the doors (20-15 bullets to destroy one)

cant you make doors next to doors?

**Comment by MLGTASTICa** [comment]
> > > make a maze where xenos can fully walk into marines can be stuck and for it to take the same hits to break as a normal wall maze?
> > 
> > 
> > Doors need adjaecent resin walls as support , ontop of it , doors do not regenerate like walls do (and they don't get a health bonus the earlier they were built either). As a marine , you could also not rush into the maze and instead wait for flamethrowers or MGs to take care of the doors (20-15 bullets to destroy one)
> 
> cant you make doors next to doors?

You can , but they're still weaker than walls?

**Comment by Lumipharon** [comment]
Doors are deliberately far weaker than walls. Strong doors means you can just spam doors and it's impossible to chase xenos without getting trapped, which is terrible for balance.

**Comment by Naaanii** [comment]
Doors are meant to be weaker, but also take longer to build and spam I guess.

Upping armour might be okay but making them the same as walls isn't as then there would be little to no downside to doors vs walls.

**Comment by Naaanii** [comment]
> Doors are meant to be weaker, but also take longer to build and spam I guess.
> 
> Upping armour might be okay but making them the same as walls isn't as then there would be little to no downside to doors vs walls.

Nevermind, just read the changes. Seems okay.

**Review by BraveMole — CHANGES_REQUESTED** [review]
I don't really get the point of that. Doors are already usefull, why would you want to buff them ? 

It can be really hard as a marine to fight against a hell maze of doors, I sure do not want to make that harder

**Comment by MLGTASTICa** [comment]
> I don't really get the point of that. Doors are already usefull, why would you want to buff them ?
> 
> It can be really hard as a marine to fight against a hell maze of doors, I sure do not want to make that harder

Doors are not usefull , they get destroyed by any gun within a few shots. Xenos that know their game litteraly do not spam doors since they are far too weak.
If the door maze thing is such a worry , this could be test-merged , but i am very confident that marines will just do the usual thing of not running into mazes, or just setting everything on fire.

**Comment by BraveMole** [comment]
> Doors are not usefull , they get destroyed by any gun within a few shots. Xenos that know their game litteraly do not spam doors since they are far too weak.

They are not spammable, that is the whole point. That is why they take more time and more plasma to build.

We tried other door buffs and everytime it made xeno spam them, this was hell to play against. No on that

**Comment by MLGTASTICa** [comment]
> > Doors are not usefull , they get destroyed by any gun within a few shots. Xenos that know their game litteraly do not spam doors since they are far too weak.
> 
> They are not spammable, that is the whole point. That is why they take more time and more plasma to build.
> 
> We tried other door buffs and everytime it made xeno spam them, this was hell to play against. No on that

Doors take 3 times more plasma to build and are twice as slow
Ontop of this , the doors themselves are still far weaker than walls.
Hell , since you even brought the "other door buffs" argument ,i checked every PR that even touched resin doors. They no longer block fire and you already made them way more expensive.  They are litteraly easy to destroy , especially with CAS and Flamers. You even tried to deny a PR that  buffed their health from 50 to 100 , because that'd make them too op ? (they litteraly die in 5 P90 shots). You should atleast gives this a chance and test-PR it for a day or 2 and see if it can really be abused to  get "easy" wins as xeno.

**Comment by FosterGrime-Heymynameslouie** [comment]
I can see door hive, and or having literally every tile of caves be filled with doors like the good old days with this pr up be pretty funny, but given how the doors cost 3x the cost of plasma compared to walls, maybe even more to balance it further, it might probably be worth a shot? 

A problem that might potentially happen is that hivelords and hiveminds can print out resin structures like no tommorow, even more so pre shutter drop causing the door hive strat to come up, akin to to the time when doors were given fast opening/close times WHICH caused the door health to be dropped to be paper thin levels to begin with.

Might be a shot in the dark but I guess a good drawback to having 60% bull and las resist is for the door to have negative flame resist? Might encourage people to bring flamers/underbarrel flamers more, or just straight up make it cost even more. Whichever seems to be the most reasonable.

TLDR: Door hive strat might come up, but due to plasma cost it can potentially be mitigated but if not, increase price even more or, suggested drawback to 60% bull and las resist to have negative flame resist.


good pr though, doors are so ass currently and its nice to see something shake up the absolute state of doors

**Comment by MereWalker** [comment]
I'm assuming this will make fire useful again? Please make it so.. I just want to clear mazes with fire.

**Comment by lKiseki** [comment]
Buffing resin door armor is likely not what should be done to them. I’d suggest making resin doors cost far less plasma to build, and be built faster. With how easily they’re cleared, this shouldn’t affect balance much.

**Comment by KrabSpider** [comment]
> Buffing resin door armor is likely not what should be done to them. I’d suggest making resin doors cost far less plasma to build, and be built faster. With how easily they’re cleared, this shouldn’t affect balance much.

Agreed. Keeping the cost and time the same (even though those are your justifications for buffing the armor values) is rather pointless imo. 

60% proj armor is also rather overkill tbh.

**Comment by BraveMole** [comment]
Closing, no maintainers thought this was a good idea
